src/components/pages/coach/Anamnesis/index.jsx

```diff
@@ -174,16 +210,17 @@ class Anamnesis extends React.Component{
     }
 
     getAnmaneseData = () => {
-        console.log(this.state.user);
+        console.log("called");
         this.props.client.query({
         query: GET_DATA,
         variables: { 
             "token": localStorage.getItem('fprint'),
```

Comment:
"fprint" doesn't seem to be the best naming for an JWT in local storage.
I would prefer to name it anonymousJWT or something along those lines,
cause it's the JWT of the anonymous user.

Reply:
I have to agree on that one. They are a remenant of darker times. 🌋